source/reference/data-types.txt

```diff
@@ -322,8 +324,8 @@ string alias ``"decimal"`` or ``19``, the numeric code for the
 
 .. _check-types-in-shell:
       
-Check Types in the ``mongo`` Shell
-----------------------------------
+Check Types in the MongoDB shell, ``mongosh``
```

Comment:
Sorry I missed this in the review. Can you please make this "MongoDB Shell" ? (title case).

Reply:
Done.